data/Templates/FhirToHL7v2/DataType/_CWE-CodedWithExceptions.liquid

```diff
@@ -41,6 +41,9 @@ https://hl7-definition.caristix.com/v2/HL7v2.8/DataTypes/CWE
 {% comment %} 4  Alternate Identifier            {% endcomment %}^
 {% comment %} 5  Alternate Text                  {% endcomment %}^
 {% comment %} 6  Name Of Alternate Coding System {% endcomment %}{{ code.version }}^
+{%- when 'HL70006' -%}
```

Comment:
Does this one need the name of the coding system (HL700006) in 3 like the above examples?

Reply:
Maybe but it was not present in the initial HL7.

https://github.com/jasonheld-msft/FHIR-Converter/blob/main/data/Templates/FhirToHL7v2/Segments/_PID.liquid#L51